src/rt.rs

```diff
@@ -16,7 +16,7 @@ use super::Result;
 type Software<'a> = Box<dyn Fn() -> Pin<Box<dyn Future<Output = Result>>> + 'a>;
 
 /// Runtime kinds.
-enum Kind<'a> {
+pub enum Kind<'a> {
```

Comment:
Did you intentionally make these types public?

Reply:
Yes, the mod isn't pub so its just visible within the crate. I needed the kind to check for any client rts.